src/app/components/auth/login/login.component.html

```diff
@@ -1 +1,36 @@
-<p>login works!</p>
+<div class="maincontainer">
+<div class="container">
+    <div class="row vh-100 d-flex align-items-center justify-content-center">
+        <div class="text-left text-mb-center col-md-9 col-lg-4 col-xl-4">
+       <img src="../../../../assets/images/logo.png" alt="logo"  class="img-fluid w140  h130 ms-3">
+       <h1 class="mb-0 pt-4 fs45 fw-300 logo">EVENTOS</h1>
+       <p class="text-white fs12 fw-300">Connecting colleagues following events</p>
+    </div>
+    <div class="col-md-8 col-lg-8 col-xl-4 ">
+        <form class="text-center px-5 py-5 pt-2 rounded-box">
+         <h2 class="fw-500 interfont fs33 text-white pt-2 pb-2">Login</h2>
+          <div class="form-group mb-4 mt-2">
+            <input type="email" id="form3Example3" class="form-control input-form-md-rounded fs12 fw-500"
+              placeholder="Email or Mobile number"  autocomplete="off"/>
+          </div>
+
+          <div class="form-group mb-2">
+            <input type="password" id="form3Example4" class="form-control input-form-md-rounded fs15 fw-500"
+              placeholder="password"  autocomplete="off"/>
+          </div>
+
+          <div class="d-inline-flex justify-content-between align-items-center text-center">
+            <a href="#!" class="text-decoration-none dark-green fs13 fw-500 interfont">Forgot password?</a>
+          </div>
+
+          <div class="text-center text-lg-start mt-0 pt-2">
+            <button type="button" class="btn btn-primary w-100 blue-light fw-500 fs15 interfont"
```

Comment:
Remove "blue-light" class and use one common class in button.scss file

Reply:
class removed